### PR TITLE
[kserve] Add KServe Testing Labels

### PR DESCRIPTION
**Pull request by drewrip**
/var matbench.download.mode=important

### CODE REVIEW

File: projects/kserve/testing/config.yaml [hunk 1]
```diff
@@ -102,7 +108,7 @@ ci_presets:
   quick_raw:
     extends: [raw, no_gpu, tgis]
 
-    tests.e2e.models: [flan-t5-small-hf-cpu, flan-t5-small-hf-cpu]
+    tests.e2e.models: [flan-t5-small-hf-cpu]
```
Comment (kpouget) [inline projects/kserve/testing/config.yaml:111]: please put the second model back before merging (having 2 recorded tests more code paths in the visualization)

Comment (drewrip) [reply projects/kserve/testing/config.yaml]: Replaced this

**Comment by kpouget** [comment]
oh, [test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift-psap_topsail/217/pull-ci-openshift-psap-topsail-main-rhoai-light/1764245156212510720/artifacts/light/test/artifacts/report_index.html)!

**Comment by kpouget** [comment]
[test passed](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/openshift-psap_topsail/217/pull-ci-openshift-psap-topsail-main-rhoai-light/1764281450363883520/artifacts/light/test/artifacts/report_index.html), merging this version :)

**Pull request merged**